documentation/identifiers.md

```diff
@@ -78,65 +88,94 @@ There are many different kinds of organisations that give, receive or benefit fr
 * Registered charities
 * Community organisations
 * Overseas organisations
+* Public bodies
 * Schools, universities and other educational establishments
 * Individuals
 
-Many organisations will have official registration numbers, assigned by official registration agencies, that can be used to identify them. For example, in the UK [Companies House](http://www.companieshouse.gov.uk) can be considered to be a registration agency that assigns unique numbers to companies. In order to uniquely identify that company you would need to know both the company number and that it came from Companies House. (Crucially, you can also look up a company, given its number, at Companies House to find out more detail about that company.) Wherever possible 360Giving encourages you to collect and record these registered identifiers (Organisation Identifiers). 
+Most organisations have some sort of official registration number that can be used to uniquely identify them and to look up their details from an official registers or public list. 
 
-Organisation Identifiers take the form:
+There are two parts to an organisation identifier:
 
->*Registration Agency Code* (dash) *Identifier*
+* **A list code** that describes the list the identifier is taken from
```

Comment:
Bob also mentioned - need consistency on how describe registers/lists. My preference is for 'list code prefix' throughout. eg
"•	A list code prefix that describes the list the identifier is taken from"

Reply:
Yeah, that would be right. I'm going to be *really* pedantic and write 
> A **list code**: a prefix that describes.... '
as in this context it's a prefix for the id, but it's still the list code itself (now that's pedantic).